Apply Fortinet recommandations

squioc opened the pull request:
Apply the recommendations from Fortinet

squioc committed:
refactor: move the connector to the top-level

squioc committed:
fix: change import

squioc committed:
fix: add logo and update info.json

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -17,15 +20,18 @@
                 "editable": true,
                 "visible": true,
-                "type": "text"
+                "type": "password",
```
This doesn't work, only type text works
```suggestion
                "type": "text",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -17,15 +20,18 @@
                 "editable": true,
                 "visible": true,
-                "type": "boolean"
+                "type": "checkbox",
```
same thing here, the type should be boolean otherwise it doesn't work
```suggestion
                "type": "boolean",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -17,15 +20,18 @@
                 "visible": true,
-                "type": "boolean"
+                "type": "checkbox",
+                "value": true,
```
```suggestion
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -146,102 +161,113 @@
                     "editable": true,
                     "type": "boolean",
-                    "name": "comments"
+                    "name": "comments",
```
```suggestion
                    "name": "include_comments",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -146,102 +161,113 @@
                     "editable": true,
                     "type": "boolean",
-                    "name": "stix"
+                    "name": "stix",
```
```suggestion
                    "name": "include_stix",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -146,102 +161,113 @@
                     "editable": true,
                     "type": "boolean",
-                    "name": "history"
+                    "name": "history",
```
```suggestion
                    "name": "include_history",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -146,102 +161,113 @@
                     "editable": true,
                     "type": "boolean",
-                    "name": "countermeasures"
+                    "name": "countermeasures",
```
```suggestion
                    "name": "include_countermeasures",
```

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update get_alert.py

Fix get_alert file errors

yosradirgham commented on sekoiaio/connector.py:
```diff
@@ -1,17 +1,18 @@
 from .update_asset import update_asset
 
-logger = get_logger("sekoiaio")
+logger = get_logger("sekoiaio-xdr")
```
```suggestion
logger = get_logger("sekoiaio")
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
-                    "editable": true,
-                    "type": "text",
-                    "name": "community_uuid"
+                    "name": "owners",
```
```suggestion
                    "name": "asset_owners",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
                     "editable": true,
                     "type": "text",
-                    "name": "keys"
+                    "name": "keys",
```
```suggestion
                    "name": "asset_keys",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
                     "editable": true,
                     "type": "text",
-                    "name": "attributes"
+                    "name": "attributes",
```
```suggestion
                    "name": "asset_attributes",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
                     "editable": true,
                     "type": "text",
-                    "name": "description"
+                    "name": "description",
```
```suggestion
                    "name": "asset_description",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
                     "editable": true,
                     "type": "text",
-                    "name": "criticity"
+                    "name": "criticity",
```
```suggestion
                    "name": "asset_criticity",
```

yosradirgham commented on sekoiaio/info.json:
```diff
@@ -266,71 +292,71 @@
                     "editable": true,
                     "type": "text",
-                    "name": "name"
+                    "name": "name",
```
```suggestion
                    "name": "asset_name",
```

yosradirgham committed:
Update sekoiaio/connector.py

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update update_asset.py

Fix action update_asset

yosradirgham committed:
Update info.json

Fix info.json for action update_asset

yosradirgham committed:
Update activate_countermeasure.py

Fix url! Unfortunately the urljoin doesn't work as expected, that's why I was obliged to remove all the urljoin I was using! Maybe it's related to the python version we use???

yosradirgham commented on sekoiaio/activate_countermeasure.py:
```diff
@@ -1,15 +1,16 @@
 from urllib.parse import urljoin
```
```suggestion
```

yosradirgham commented on sekoiaio/deny_countermeasure.py:
```diff
@@ -1,15 +1,19 @@
     Deny a countermeasure
     """
-    url: str = urljoin(OPERATION_CENTER_BASE_URL, f"countermeasures/{params['countermeasure_uuid']}/deny")
+    url: str = urljoin(
+        OPERATION_CENTER_BASE_URL,
+        f"countermeasures/{params['countermeasure_uuid']}/deny",
+    )
```
```suggestion
    url: str = f"{OPERATION_CENTER_BASE_URL}/countermeasures/{params['countermeasure_uuid']}/deny"
```

yosradirgham commented on sekoiaio/deny_countermeasure.py:
```diff
@@ -1,15 +1,19 @@
 from urllib.parse import urljoin
```
```suggestion
```

yosradirgham committed:
Update sekoiaio/activate_countermeasure.py

yosradirgham committed:
Update sekoiaio/deny_countermeasure.py

yosradirgham committed:
Update sekoiaio/deny_countermeasure.py
